src/pages/Signup/index.js

```diff
@@ -25,6 +24,7 @@ const Signup = () => {
 
   const schema = z
     .object({
+      //name: z.string().name({ message: t('signup.errors.nameMsg') }),
```

Comment:
What do you think about removing this commented line?

Reply:
I strongly agree! This comment came with the template repo but I should have deleted it.